### ISSUE TITLE
Possible bug in MSE 

### ISSUE THREAD

**Issue by AakashKumarNain** [keras-team/keras-core#8]
Shouldn't the `mean_squared_error` return the mean values?  The loss function currently returns `(y_true - y_pred) ** 2` but shouldn't it be `ops.mean((y_true - y_pred) ** 2)` instead? An example to illustrate the same

```python
y_true = np.array([[0., 1.], [0., 0.]])
y_pred = np.array([[1., 1.], [1., 0.]])

loss = MeanSquaredError()
loss(y_true, y_pred) # Returns 1.0 but should return 0.5
```
